mailinglogger/MailingLogger.py

```diff
@@ -39,7 +40,8 @@ def __init__(self,
                  headers=None,
                  template=None,
                  charset='utf-8',
-                 content_type='text/plain'):
+                 content_type='text/plain',
+                 secure=None):

```

Comment:
because this is not the signature of use_tls. use_tls is a boolean, secure is a tuple
i called it secure because it is called secure in SMTPHandler (parent class)